client/app/hosting/multisite/MULTISITE.html

```diff
@@ -181,9 +181,18 @@
                 <button
                     type="button"
                     class="btn btn-block btn-default text-wrap"
-                    data-ng-bind="sslCertificate ? ('hosting_ssl_regenerate_title_button' | translate) : ('hosting_dashboard_service_order_ssl' | translate)"
-                    data-ng-click="setAction(sslCertificate ? 'ssl/regenerate/hosting-ssl-regenerate' : 'ssl/order/hosting-order-ssl')"
-                    data-ng-if="!hosting.hasHostedSsl && !(sslCertificate.status === 'deleting' || sslCertificate.status === 'regenerating' || sslCertificate.status === 'creating')">
+                    data-translate="hosting_dashboard_service_order_ssl"
+                    data-ng-click="setAction('ssl/order/hosting-order-ssl')"
+                    data-ng-if="!sslCertificate || !(sslCertificate.status === 'deleting' || sslCertificate.status === 'regenerating' || sslCertificate.status === 'creating')">
```

Comment:
Could you have this in a function please ? This could make it easier to maintain :)

Reply:
This is done